pkg/fileservice/config.go

```diff
@@ -33,14 +33,14 @@ const (
 type Config struct {
 	// Name name of fileservice, describe what an instance of fileservice is used for
 	Name string `toml:"name"`
-	// Backend fileservice backend. [MEM|DISK|S3|MINIO]
+	// Backend fileservice backend. [MEM|DISK|S3|ETL]
```

Comment:
Why change MINIO to ETL? These names are defined in https://github.com/matrixorigin/matrixone/blob/319b4c7e093db93a8a25c9e3e597f4068d35f215/pkg/fileservice/config.go#L24